### PR TITLE
Newsletter implementation

### PR DESCRIPTION
**Pull request by JustinCast**
### What does this PR do?
- Implement a basic newsletter connected to substack and disable not working-existent form for newsletter

### CODE REVIEW

**Comment by JustinCast** [comment]
@xavier506 @edgarithm

File: src/components/Layout/Footer/index.tsx [hunk 1]
```diff
@@ -44,10 +44,10 @@ const Footer: React.FC<FooterProps> = ({ scrollTop }) => {
             <Grid container>
               <Grid item md={12} xs={12}>
                 <Box className={classes.floatBox}>
-                  <Typography variant='h3'>
+                  {/* <Typography variant='h3'>
```
Comment (xavier506) [inline src/components/Layout/Footer/index.tsx:47]: why do we want to keep the code comments?

Comment (JustinCast) [reply src/components/Layout/Footer/index.tsx:47]: Let's remove them 👌🏻

**Review by xavier506 — APPROVED** [review]
LGTM

**Review by AngeloCG97 — APPROVED** [review]
LGTM

**Comment by xavier506** [comment]
Created a release tag,  build is here:

https://github.com/edenia/asoblockchain.org/actions/runs/2741493587

**Comment by JustinCast** [comment]
> Created a release tag, build is here:
> 
> https://github.com/edenia/asoblockchain.org/actions/runs/2741493587

Dope!, thanks

**Pull request merged**